dcos/util.py

```diff
@@ -101,17 +101,23 @@ def read_file(path):
         return file_.read()
 
 
+CONFIG = None
```

Comment:
I've made the user config into a singleton, because it simplifies a lot of things, for example the slave's fetching their state.json.